nodejs/week3/warmUp/index.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.json());
+
+// Calculator using params
+app.get('/calculator/:param', (req, res) => {
+	const { param } = req.params;
+	const { firstParam } = req.query;
+	const { secondParam } = req.query;
+	const allNumbers = [];
```

Comment:
AllNumbers is vague.

Reply:
Like it takes in all the numbers I get from firstParam & secondParam

Reply:
Maybe `queryparameterNumbers`